src/DearImGui/Enums.hs

```diff
@@ -0,0 +1,14 @@
+{-# LANGUAGE DerivingStrategies #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE PatternSynonyms #-}
+{-# LANGUAGE TemplateHaskell #-}
+
+module DearImGui.Enums where
+
+-- dear-imgui-generator
+import DearImGui.Generator
+  ( declareEnumerations )
+
+--------------------------------------------------------------------------------
+
+declareEnumerations
```

Comment:
Is it possible to print what declareEnumerations output? If so, would it be ok to add some instructions for contributor not familiar with template haskell?

Reply:
`haskell-language-server` has a "splice" plugin which can be used to see the contents of the splice. You can also pass the `-ddump-splices` flag to GHC, which will output the result. However in this case I think it would suffice to read the documentation generated by haddock.